### DISCUSSION TITLE
[Feature Request] Custom CSS Options

### DISCUSSION THREAD

**Discussion by WillMontgomery** [diced/zipline#352] [Ideas]
It would be neat to have custom CSS options available. I'm not sure how this would be implemented, but an idea I had would be in the account settings page to add a field where the user could enter hex values for colors for each element, maybe select from a few fonts, or perhaps in the form of pre-selected color scheme options.

**Comment by TacticalTechJay** [comment]
If you'd like, you could fork zipline and make yourself a theme! It's defined over in `src/lib/themes`.

**Comment by diced** [comment]
Yes, as @TacticalTechJay said you can do that, but I will **not** be adding custom css themes to Zipline.
